src/pattern-library/components/patterns/ButtonPatterns.js

```diff
@@ -81,6 +81,48 @@ export default function ButtonPatterns() {
           </PatternExample>
         </PatternExamples>
 
+        <h3>Dark variant</h3>
+        <p>
+          This variant is for use on darker (light grey vs. white) backgrounds.
+          Note that this button, unlike other <code>IconButton</code>s, has a
+          background color. This is to allow for a use case in which the dark{' '}
+          <code>IconButton</code> is initially fixed on a grey background but
+          floats on top of content when scrolled.
+        </p>
+
+        <PatternExamples>
+          <PatternExample
+            details="Basic usage"
+            style={{ backgroundColor: '#ececec' }}
```

Comment:
You could create a variable: `const backgroundColor = '#ececec';` and use to replace all the lines:
```suggestion
            style={{ backgroundColor }}
```

Reply:
It's not DRY, but I'd like to leave it as-is as it's easy to understand and scan, and we may end up with examples on this page that use a different background color.